internal/dispatch/combined/combined.go

```diff
@@ -196,51 +196,51 @@ func NewDispatcher(options ...Option) (dispatch.Dispatcher, error) {
 		return nil, err
 	}
 
-	chunkSize := opts.dispatchChunkSize
-	if chunkSize == 0 {
-		chunkSize = 100
-		log.Warn().Msgf("CombinedDispatcher: dispatchChunkSize not set, defaulting to %d", chunkSize)
-	}
+	var redispatch dispatch.Dispatcher
 
-	cts := caveattypes.TypeSetOrDefault(opts.caveatTypeSet)
+	if opts.upstreamAddr == "" {
```

Comment:
FYI major change on the setup. Notice how before, `redispatch`'s content was getting overwritten on line 307